tcclient/http.go

```diff
@@ -13,7 +13,7 @@ import (
 	"reflect"
 
 	"github.com/taskcluster/httpbackoff"
-	"github.com/tent/hawk-go"
+	hawk "github.com/tent/hawk-go"
```

Comment:
not sure how it worked without this! package names can't contain `-` character so we have to give it a name.